lib/compiler-singlepass/src/emitter_x64.rs

```diff
@@ -946,6 +946,43 @@ impl EmitterX64 for AssemblerX64 {
         Ok(())
     }
 
+    fn arch_has_xzcnt(&self) -> bool {
+        match &self.target {
+            Some(target) => {
+                target.cpu_features().contains(CpuFeature::LZCNT)
+                    && target.cpu_features().contains(CpuFeature::BMI1)
```

Comment:
More curiosity than comment: 

I guess `xzcnt` means TZCOUNT and LZCOUNT.
TZCOUNT is part of BMI1, right?

Is there some good resource to look up CPUID mappings , other than the reference manuals? :D

Reply:
> TZCOUNT is part of BMI1, right?

Yes

> Is there some good resource to look up CPUID mappings , other than the reference manuals? :D

Not that I am aware off. I use Intel manuel and various web search when I have question like that.